bower.json

```diff
@@ -12,7 +12,8 @@
     "toastr": "^2.1.2",
     "hint.css": "^2.3.2",
     "jquery.tagsinput": "^1.3.6",
-    "loaders.css": "^0.1.2"
+    "loaders.css": "^0.1.2",
+    "jquery-mockjax": "~2.2.1"
```

Comment:
`jquery-mockjax` sounds like something used only for testing. Could I suggest moving it to `devDependencies` instead of `dependencies`?

See: http://stackoverflow.com/questions/19339227/bower-and-devdependencies-vs-dependencies